### DISCUSSION TITLE
Problem transferring photos from iOS to Android (file name change)

### DISCUSSION THREAD

**Discussion by Yuslendo** [localsend/localsend#19] [Discussion (English, International)]
When transferring photos from my iPhone X to my Samsung Galaxy Tab S8+ (SM-X800) tablet, the filenames turn into random numbers. I don't have this problem when transferring to windows device. I want to keep the files with their original name. I hope this issue will be fixed in the next updates. I can give more information about the problem if wanted.
![Ekran görüntüsü_20230122_174841](https://user-images.githubusercontent.com/123311399/213922928-0321946b-4124-44cb-8864-db4b8a0ac338.png)
![FullSizeRender](https://user-images.githubusercontent.com/123311399/213922930-3f8b8a6d-6a92-40ac-9607-c6c782fc7f76.jpg)
![Screenshot_20230122_175241_My Files](https://user-images.githubusercontent.com/123311399/213922931-d43d845e-b639-469b-814f-c67df5e04558.jpg)

**Comment by Tienisto** [comment]
What happens if you disable "Save media to gallery"?

**Reply by Yuslendo** [reply]
It worked. Photos also displaying in the gallery.
![Screenshot_20230125_060406_My Files](https://user-images.githubusercontent.com/123311399/214471490-c1395354-f5e1-4e49-be6f-21fe7e744de2.jpg)
![Screenshot_20230125_060330_Gallery](https://user-images.githubusercontent.com/123311399/214471495-27e30190-d642-40f3-8524-70e1fc33af18.jpg)
![FullSizeRender](https://user-images.githubusercontent.com/123311399/214471497-614882d9-99be-4022-9ef3-d103fcd2f98b.jpg)

**Comment by Tienisto** [comment]
Since `v1.7.0`, you can keep "Save media to gallery" on. This is preferred otherwise, your Downloads folder will get full very quickly.

**Reply by Yuslendo** [reply]
Thanks for improvement!
